### PR TITLE
v7.23.0 – gh-pages dependency changed and lintModules flag added

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
Can now lint sub-modules Sass (for local development) by using the `--lintModules` flag.

### Changed
- Dependency on gulp-gh-pages removed in favour of our own forked version (as the maintainer had moved it to Gulp 4, which hasn't been released).
- Tests updated in line with config changes to CSS lintPaths

### Removed
- Gemnasium removed from README

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,21 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v7.23.0
+------------------------------
+*August 17, 2018*
```
Comment (xander-marjoram) [inline CHANGELOG.md]: 📆

File: README.md [hunk 2]
```diff
@@ -899,6 +898,12 @@ The following options are also present in the config but cannot be overridden.
 
   Set to the opposite value of `isProduction`.
 
+- #### `lintModules`
+
+  Type: `array`
```
Comment (xander-marjoram) [inline README.md]: `boolean`?

File: CHANGELOG.md (hunk 1 above)
Comment (ashleynolan) [reply CHANGELOG.md]: 🎈

File: README.md (hunk 2 above)
Comment (ashleynolan) [reply README.md]: Yep – fixed

**Comment by ashleynolan** [comment]
@DamianMullins @sonnyprince Have just pushed small change (based on comments from @xander-marjoram) – any chance you could re-tick if you're happy?

**Pull request merged**